### PR TITLE
Adding of Tag and TagManager class

### PR DESCRIPTION
**Pull request by Chongjx**
#26 

*Creation of Tag
*Deletion of Tag
*Tag notes

### CODE REVIEW

File: src/test/java/seedu/duke/command/TagCommandTest.java [hunk 1]
```diff
@@ -7,8 +7,4 @@
     @Test
     void execute() {
```
Comment (R-Ramana) [inline src/test/java/seedu/duke/command/TagCommandTest.java:8]: If it is a test method should we not follow the following naming conventions?

`whatIsBeingTested_descriptionOfTestInputs_expectedOutcome` 
OR
`testMethodName`

**Review by Nazryl — APPROVED** [review]
Excellent work. Love the implement on coloured tags.

**Review by R-Ramana — COMMENTED** [review]
Great work! Just one clarification which you can see below this comment!

File: src/test/java/seedu/duke/command/TagCommandTest.java (hunk 1 above)
Comment (Chongjx) [reply src/test/java/seedu/duke/command/TagCommandTest.java:8]: yup it should! I just generate a template test class therefore it consists of the execute function inside. All the test functions should be renamed accordingly.

Comment (Nazryl) [reply src/test/java/seedu/duke/command/TagCommandTest.java:8]: He haven't start on it. He created the classes as a template first I think.

File: src/main/java/seedu/duke/command/FindCommand.java [hunk 2]
```diff
@@ -7,6 +7,8 @@
 
     public static final String COMMAND_WORD = "find-n";
 
+    public static final String MESSAGE_USAGE = "";
```
Comment (R-Ramana) [inline src/main/java/seedu/duke/command/FindCommand.java:10]: Maybe can update this to COMMAND_USAGE to be clearer on what this String is for :)

**Review by R-Ramana — APPROVED** [review]

**Pull request merged**